maps/src/main/res/layout/offline_map_layers_picker.xml

```diff
@@ -0,0 +1,123 @@
+        android:src="@drawable/ic_outline_info_24"
+        app:layout_constraintStart_toStartOf="@id/message"
+        app:layout_constraintTop_toBottomOf="@id/message"
+        app:tint="?colorPrimary" />
+
+    <com.google.android.material.textview.MaterialTextView
+        android:id="@+id/mbtiles_info_icon_text"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/margin_standard"
+        android:text="@string/get_help_with_reference_layers"
+        android:textAppearance="?textAppearanceLabelLarge"
+        android:textColor="?colorAccent"
+        app:layout_constraintBottom_toBottomOf="@id/mbtiles_info_icon"
+        app:layout_constraintEnd_toEndOf="@id/message"
+        app:layout_constraintStart_toEndOf="@id/mbtiles_info_icon"
+        app:layout_constraintTop_toTopOf="@id/mbtiles_info_icon" />
+
+    <androidx.constraintlayout.widget.Group
+        android:id="@+id/mbtiles_info_group"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:visibility="visible"
+        app:constraint_referenced_ids="mbtiles_info_icon,mbtiles_info_icon_text" />
+
+    <View
+        android:id="@+id/top_divider"
+        android:layout_width="match_parent"
+        android:layout_height="1dp"
+        android:background="@drawable/list_item_divider"
+        android:layout_marginTop="@dimen/margin_standard"
+        app:layout_constraintTop_toBottomOf="@id/mbtiles_info_icon" />
+
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/layers"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:visibility="gone"
+        app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
+        app:layout_constraintBottom_toTopOf="@id/bottom_divider"
+        app:layout_constraintHeight_default="wrap"
+        app:layout_constraintTop_toBottomOf="@id/top_divider"
+        tools:visibility="visible" />
+
+    <com.google.android.material.progressindicator.CircularProgressIndicator
+        android:id="@+id/progress_indicator"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginVertical="@dimen/margin_small"
+        android:indeterminate="true"
+        app:layout_constraintBottom_toTopOf="@id/bottom_divider"
+        app:layout_constraintEnd_toEndOf="@id/message"
+        app:layout_constraintStart_toStartOf="@id/message"
+        app:layout_constraintTop_toBottomOf="@id/top_divider" />
+
+    <View
+        android:id="@+id/bottom_divider"
+        android:layout_width="match_parent"
+        android:layout_height="1dp"
+        android:background="@drawable/list_item_divider"
+        app:layout_constraintBottom_toTopOf="@id/save" />
+
+    <org.odk.collect.androidshared.ui.multiclicksafe.MultiClickSafeMaterialButton
+        android:id="@+id/save"
+        style="?materialButtonStyle"
```

Comment:
@seadowg please take a look at this button. The styling does not work well. If it was an activity everything would be fine (for example we use the same styling in the crash-handler module and there it works well). There is something wrong with using it in fragments in separate modules I guess. Do you maybe know what is wrong?

Reply:
Oh wow that's weird! It looks to me like bottom sheets might get their own theme overlay set on them (although I haven't looked into it deeply). The hint for that is that you can set style to `@style/Widget.Material3.Button` and get what you'd expect, but using `?materialButtonStyle` is just giving us a bolder text button. My guess is that bottom sheets have an overlay theme that overrides `?materialButtonStyle` and that we might need to customise and set our own theme for them (like we do with dialogs).

Reply:
Yeah, it looks like we need to set our own theme for bottom sheets.